docs/proposed/psa-driver-interface.md

```diff
@@ -461,6 +459,7 @@ The `"key_context"` property in the [driver description](#driver-description-top
 * `"symmetric_factor"` (integer or string, optional): every key context for a symmetric key includes this many times the key size. If omitted, this value defaults to 0.
 * `"store_public_key"` (boolean, optional): If specified and true, for a key pair, the key context includes space for the public key. If omitted or false, no additional space is added for the public key.
 * `"size_function"` (string, optional): the name of a function that returns the number of bytes that the driver needs in a key context for a key. This may be a pointer to function. This must be a C identifier; more complex expressions are not permitted. If the core uses this function, it supersedes all the other properties.
+* `"builtin_key_size"` (integer or string, optional): If specified, this overrides all other methods (including the `"size_function"` entry point) to determine the size of the key context for [built-in keys](#built-in-keys). This allows drivers to efficiently represent application keys as wrapped key material, but built-in keys by an internal identifer that takes up less space.
```

Comment:
I think the description for `size_function` needs to be updated : "If the core uses this function, it supersedes all the other properties _except for built-in keys if `builtin_key_size` is used." Otherwise we have two things that say they supersede/override all the others.

Reply:
This actually seems clear as it stands to me, although Manuel's suggestion does seem like a further improvement. So I'm mildly in favour of noting the exception in the size_function paragraph as suggested.

Reply:
@mpg I am on the verge of merging this PR but as you asked a change at some point (see above), I just want to be sure this is ok with you eventually. Could you have a look?

Reply:
Just had a look, I'm happy with the change made by Gilles.